### ISSUE TITLE
Host announcement here

### ISSUE THREAD

**Issue by maurolepore** [2DegreesInvesting/r2dii.match#186]
See https://github.com/2DegreesInvesting/r2dii/issues/13

**Comment by maurolepore** [comment]
* [ ] `use_article("r2dii.match-0-0-3.Rmd")` to add the article's yaml header and setup chunk.
* [ ] Write the article.
* [ ] `rmarkdown::render("vignettes/r2dii.match-0-0-3.Rmd", "md_document")`
* [ ] Commit
* [ ] In the .Rmd file, replace the source with the output from the .md file. 
* [ ] Remove the .md
* [ ] Ensure the article is added to buildignore and R CMD check passes.
* [ ] Link the article to News in _pkgdown.yml, e.g.:
```
  - text: "News"
    menu:
    - text: "Release notes"
    - text: "r2dii.data 0.0.3"
      href: articles/r2dii-data-0-0-3.html
```

**Comment by maurolepore** [comment]
Closed by #187
